src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -8,7 +8,7 @@ import { DEFAULT_WAIT_FOR_ELEMENT_TO_BE_VISIBLE_TIMEOUT_MS } from 'tests/electro
 import { AutomatedChecksViewController } from './automated-checks-view-controller';
 
 export class AppController {
-    private client: SpectronAsyncClient;
+    public client: SpectronAsyncClient;
```

Comment:
I didn't mean to change this to private in https://github.com/microsoft/accessibility-insights-web/pull/2596. At one point we wanted to restrict access to avoid exposing the raw webdriverio typings, but the SpectronAsyncClient interface resolves that issue.